stdlib/array/from_group_test.flux

```diff
@@ -18,53 +19,55 @@ data = "
 ,,0,m0,f0,tagvalue,2018-12-19T22:14:10Z,true
 ,,0,m0,f0,tagvalue,2018-12-19T22:14:20Z,true
 "
-input = array.from(
-    rows: [
-        {
-            _measurement: "m0",
-            _field: "f0",
-            t0: "tagvalue",
-            _time: 2018-12-19T22:13:30Z,
-            _value: false,
-        },
-        {
-            _measurement: "m0",
-            _field: "f0",
-            t0: "tagvalue",
-            _time: 2018-12-19T22:13:40Z,
-            _value: true,
-        },
-        {
-            _measurement: "m0",
-            _field: "f0",
-            t0: "tagvalue",
-            _time: 2018-12-19T22:13:50Z,
-            _value: false,
-        },
-        {
-            _measurement: "m0",
-            _field: "f0",
-            t0: "tagvalue",
-            _time: 2018-12-19T22:14:00Z,
-            _value: false,
-        },
-        {
-            _measurement: "m0",
-            _field: "f0",
-            t0: "tagvalue",
-            _time: 2018-12-19T22:14:10Z,
-            _value: true,
-        },
-        {
-            _measurement: "m0",
-            _field: "f0",
-            t0: "tagvalue",
-            _time: 2018-12-19T22:14:20Z,
-            _value: true,
-        },
-    ],
-)
-pass = (tables=<-) => tables
-    |> group(columns: ["_measurement", "_field", "t0"])
+input =
+    array.from(
```

Comment:
I would have expected `input = array.from(` to fit on a single line. Is the reason it is not formatted unto a single line because the `hang_doc` logic doesn't have explicit support for CallExpressions? 

Not worried about the change, but trying to understand why a newline was needed in this case.

Reply:
I am going to play around with the branch to see if I can make this change, mostly just to make sure I grasp the new framework.